spotinst/commons/common_organization_user.go

```diff
@@ -0,0 +1,117 @@
+package commons
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/organization"
+	"log"
+)
+
+const (
+	OrgUserResourceName ResourceName = "spotinst_organization_user"
+)
+
+var OrgUserResource *OrgUserTerraformResource
+
+type OrgUserTerraformResource struct {
+	GenericResource
+}
+
+type OrgUserWrapper struct {
+	orgUser *organization.User
+}
+
+func NewOrgUserResource(fieldsMap map[FieldName]*GenericField) *OrgUserTerraformResource {
+	return &OrgUserTerraformResource{
+		GenericResource: GenericResource{
+			resourceName: OrgUserResourceName,
+			fields:       NewGenericFields(fieldsMap),
+		},
+	}
+}
+
```

Comment:
let's the hierarchy same across all resources: OnCreate, OnRead, OnUpdate

Reply:
changed in same order